src/analysis/normed_space/riesz_lemma.lean

```diff
@@ -0,0 +1,30 @@
+import analysis.normed_space.basic
+import topology.metric_space.hausdorff_distance
+
+variables {k : Type*} [normed_field k]
```

Comment:
Could you use the fancy `𝕜` instead of `k` in all your file, to avoid mistaking it for an integer?

Reply:
Did that (c1ad053534bbd0fef41b2e7e7a7e38e5fa44e27e), but by copying the fancy k's over from another file. Lean mode is telling me that it `can't be input by the current input method.`